src/components/FlowRunsAccordionHeader.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div :id="id" class="flow-runs-accordion-header">
+    <div class="flow-runs-accordion-header__content">
+      <p-link :to="routes.flow(flow.id)" class="flow-runs-accordion-header__name">
+        {{ flow.name }}
+      </p-link>
+      <span class="flow-runs-accordion-header__time">
+        {{ lastRunTime }}
+      </span>
+    </div>
+    <p-button inset size="sm" :aria-controls="content" @click="toggle">
+      {{ count }}
+      <p-icon size="small" icon="ChevronDownIcon" class="flow-runs-accordion-header__icon" :class="classes.icon" />
+    </p-button>
+  </div>
+</template>
+
+<script lang="ts" setup>
+  import { useNow, useSubscription } from '@prefecthq/vue-compositions'
+  import { computed } from 'vue'
+  import { useLastFlowRun } from '@/compositions/useLastFlowRun'
+  import { useWorkspaceApi } from '@/compositions/useWorkspaceApi'
+  import { useWorkspaceRoutes } from '@/compositions/useWorkspaceRoutes'
+  import { FlowRunsFilter } from '@/models/Filters'
+  import { Flow } from '@/models/Flow'
+  import { formatDateTimeRelative } from '@/utilities/dates'
+
+  const props = defineProps<{
+    flow: Flow,
+    id: string,
```

Comment:
What's with the `id` prop and attr? I can't tell exactly what that id will be but if it's coming from the API it has a large chance of being an invalid CSS selector since selectors can't start with numbers.

Reply:
I think this is a mistake I made in PAccordion shining through naming wise. PAccordion has a slot for the section header and it binds the ids for the heading and content which need to be bound for accessibility. So this id is coming straight from PAccordion. However you are correct that since the accordion sections are uuids these might be problematic. Though since they are not being used for css I think this should be fine

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id

Reply:
Hmmm ok - I don't have another solution outside of changing that prop so let's address later. I think Vue uses entirely data attributes when juggling selectors so i think we're probably in the clear since we do very little of that